app/controllers/likes_controller.rb

```diff
@@ -4,7 +4,7 @@ class LikesController < ApplicationController
   def create
     @like = Like.new(author_id: current_user.id, post_id: params[:post_id])
     @like.save
-    redirect_to user_post_path(User.find(params[:user_id]), @post)
+    redirect_to user_post_path(User.find(params[:user_id]), @post), notice: 'You have successfully liked this post'
   end
```

Comment:
- [ ] posts#create, comments#create, and likes#create actions are all expected to use flash messages for "successful and not successful path". That message is rendered in the view. Make the changes in other files where applicable.